src/styles/blocks/stars.scss

```diff
@@ -0,0 +1,16 @@
+.card__rating-stars {
+  display: flex;
+}
+
+.stars__star {
+  width: 16px;
+  height: 16px;
+  background-image: url("../images/star-active.svg");
+  margin-right: 4px;
+  background-position: 50%;
+  background-repeat: no-repeat;
```

Comment:
Please try to use loop for indicate gold stars
![image](https://user-images.githubusercontent.com/54936044/219612715-4d70701f-b998-4a83-bc42-b7ca9f78e5f9.png)